README.md

```diff
@@ -1,11 +1,37 @@
+
 # cachedcomplete
+
 Fast semi-static autocomplete for python
```

Comment:
Maybe define what is "semi-static"?

Reply:
Or maybe I don't know enough about completers :(

Reply:
> Maybe define what is "semi-static"?

Or maybe I don't know enough about completers 😞

Reply:
> > Maybe define what is "semi-static"?
> 
> Or maybe I don't know enough about completers 😞

This line should be updated to match the new description.
But I don't think the top of the README is the place for definitions :/